save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/RegistrationView.kt

```diff
@@ -221,13 +221,16 @@ class RegistrationView : AbstractView<RegistrationProps, RegistrationViewState>(
         "PARAMETER_NAME_IN_OUTER_LAMBDA",
     )
     private fun ChildrenBuilder.renderInputForm() {
+        // google does not provide us login, only e-mail, so need to trim everything after "@gmail"
+        val rawInput = state.fieldsMap[InputTypes.USER_NAME] ?: ""
```

Comment:
"" means guest? do we have some data class with such field (`GUEST`)

Reply:
why not `.orEmpty()`?

Reply:
very good suggestion, please create an issue for diktat @sanyavertolet 
We have a thousand of places where it is in save, will change everywhere

Reply:
> very good suggestion, please create an issue for diktat @sanyavertolet
> We have a thousand of places where it is in save, will change everywhere

@sanyavertolet I will create

Reply:
https://github.com/saveourtool/diktat/issues/1710